synapse/handlers/deactivate_account.py

```diff
@@ -129,6 +133,39 @@ def deactivate_account(self, user_id, erase_data, id_server=None):
 
         return identity_server_supports_unbinding
 
+    @defer.inlineCallbacks
+    def _reject_pending_invites_for_user(self, user_id):
+        """Reject pending invites addressed to a given user ID.
+
+        Args:
+            user_id (str): The user ID to reject pending invites for.
+        """
+        user = UserID.from_string(user_id)
+        pending_invites = yield self.store.get_invited_rooms_for_user(user_id)
+
+        logger.info(pending_invites)
+
+        for room in pending_invites:
+            try:
+                yield self._room_member_handler.update_membership(
+                    create_requester(user),
+                    user,
+                    room.room_id,
+                    "leave",
+                    ratelimit=False,
+                    require_consent=False,
+                )
+                logger.info(
+                    "Rejected invite for user %r in room %r", user_id, room.room_id
+                )
+            except Exception:
+                logger.exception(
+                    "Failed to reject invite for user %r in room %r:"
```

Comment:
```suggestion
                    "Failed to reject invite for user %r in room %r."
```

Reply:
Nope, the string continues on the following line. It's a copy of https://github.com/matrix-org/synapse/blob/873fe7883cf0d7cf5346a9a55d40967a35848e33/synapse/handlers/deactivate_account.py#L225